www/themes/borg/css/node-project.css

```diff
@@ -212,13 +212,19 @@ article.node-project-release.view-mode-project-search article.view-mode-teaser {
 .node-project .view-project-release-download-table th,
 .node-project .view-project-release-download-table td {
   padding: 5px .8em .0 0;
+}
+.node-project .view-project-release-download-table th {
```

Comment:
All these CSS changes are going to be (accidentally) reverted on the next update of the borg theme if they aren't also repeated in https://github.com/backdrop-contrib/borg. 

If they are specific to this site only, they should be moved into the `backdropcms` theme instead.